### PR TITLE
Lime-993 dl

### PR DESCRIPTION
**Pull request by clsava**
Split DL tests between english and welsh
Split DL tests between dva, dvla and common

### CODE REVIEW

File: acceptance-tests/src/test/resources/features/UKDrivingLicence/English/DVLA/DVLACRIAPI.feature [hunk 1]
```diff
@@ -1,7 +1,6 @@
-@drivinglicence_CRI_API
+drivinglicence_CRI_API
```
Comment (gds-jbentham) [inline acceptance-tests/src/test/resources/features/UKDrivingLicence/English/DVLA/DVLACRIAPI.feature:1]: @clsava missing an @ which is whats failing the build

**Review by gds-jbentham — APPROVED** [review]

File: acceptance-tests/src/test/resources/features/UKDrivingLicence/English/Common/DLFullJourneyCommon.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Feature: Prove Your Identity Full Journey Common
```
Comment (gds-jbentham) [inline acceptance-tests/src/test/resources/features/UKDrivingLicence/English/Common/DLFullJourneyCommon.feature:1]: I do not believe we actually need these tests there just covering the same functionality as the tests above

File: acceptance-tests/src/test/resources/features/UKDrivingLicence/English/DVA/DVACRIAPI.feature [hunk 3]
```diff
@@ -0,0 +1,70 @@
+@drivinglicence_CRI_API
```
Comment (gds-jbentham) [inline acceptance-tests/src/test/resources/features/UKDrivingLicence/English/DVA/DVACRIAPI.feature:1]: This probably doesnt need to be in an english file as its got no concept of locales its direct to the api

File: acceptance-tests/src/test/resources/features/UKDrivingLicence/English/DVLA/DVLACRIAPI.feature [hunk 4]
```diff
@@ -1,7 +1,6 @@
 @drivinglicence_CRI_API
```
Comment (gds-jbentham) [inline acceptance-tests/src/test/resources/features/UKDrivingLicence/English/DVLA/DVLACRIAPI.feature:1]: Same as DVA files structure is good but probably doesnt need english

File: acceptance-tests/src/test/resources/features/UKDrivingLicence/DVA/DVADrivingLicence.feature [hunk 5]
```diff
@@ -180,6 +180,42 @@ Feature: DVA Driving Licence Test
...
+  @DVADrivingLicence_test @build
+  Scenario Outline: DVAError tab title validation
+    Given I click on DVA radio button and Continue
+    When I should be on the page DVA Enter your details exactly as they appear on your UK driving licence - Prove your identity - GOV.UK
+    And I check the page title is Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
+    Then User enters DVA data as a <DVADrivingLicenceSubject>
+    And User clicks on continue
+    Then I check the page title is Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
+    And The test is complete and I close the driver
+    Examples:
```
Comment (ElliotMurphyGDS) [inline acceptance-tests/src/test/resources/features/UKDrivingLicence/DVA/DVADrivingLicence.feature:198]: Should this not really live in the FE validation/validation-regression tests?

File: acceptance-tests/src/test/resources/features/UKDrivingLicence/DVLA/DVLADrivingLicence.feature [hunk 6]
```diff
@@ -171,6 +171,44 @@ Feature: Driving Licence Test
...
+    Given I click on DVLA radio button and Continue
+    When I should on the page DVLA Enter your details exactly as they appear on your UK driving licence - Prove your identity - GOV.UK
+    And I check the page title is Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
+    Then User enters DVLA data as a <DrivingLicenceSubject>
+    And User clicks on continue
+    And I check the page title is Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
+    And The test is complete and I close the driver
+    Examples:
+      |DrivingLicenceSubject             |
+      |NoLastName   |
```
Comment (ElliotMurphyGDS) [inline acceptance-tests/src/test/resources/features/UKDrivingLicence/DVLA/DVLADrivingLicence.feature:191]: Same with this, is this not testing validation scenarios? Or is the idea that validation is off so this is testing retry logic when bad request data is sent?

File: acceptance-tests/src/test/resources/features/UKDrivingLicence/DVA/DVADrivingLicence.feature (hunk 5 above)
Comment (gds-jbentham) [reply acceptance-tests/src/test/resources/features/UKDrivingLicence/DVA/DVADrivingLicence.feature:198]: Yeah but I would take that on another ticket. This is a known issue

**Review by gds-jbentham — APPROVED** [review]

**Pull request merged**